pkg/arvo/app/chat-hook.hoon

```diff
@@ -81,8 +83,14 @@
     =/  old  !<(versioned-state old-vase)
     =|  cards=(list card)
     |-
-    ?:  ?=(%6 -.old)
+    ?:  ?=(%7 -.old)
       [cards this(state old)]
+    ?:  ?=(%6 -.old)
+      =.  cards
+        %+  weld  cards
+        ^-  (list card)
+        [%pass /s %agent [our.bol %chat-hook] %poke %noun !>(%fix-out-of-sync)]~
```

Comment:
This continues to weird me out. Decoupling into a separate event like this means that the OTA succeeds even if this horribly crashes or w/e.

Reply:
Is this a "don't crash if we're OTAing during a next-sys OTA" thing?

Reply:
It would only crash if you scried, right? This doesn't scry.

Reply:
A poke to self happens in the same event. If this set a timer, it would affect OTAs as you describe.

FWIW, I'm also not a fan of the poke-to-self pattern in general)